AltexSoft lab 1/scss/_media-query.scss

```diff
@@ -0,0 +1,337 @@
+
+            .accordion-note-block {
+              padding: 60px 0 0 0;
+
+              .accordion-note {
+                font-size: 13px;
+              }
+            }
+          }
+        }
+
+        .accordion-btn {
+          height: 148.5px;
+
+          &.active {
+            .accordion-title {
+              font-size: 42px;
+              line-height: 54px;
+              max-width: 359px;
+            }
+          }
+
+          .accordion-title {
+            font-size: 24px;
+            line-height: 36px;
+          }
+
+          .accordion-number {
+            font-size: 110px;
+            line-height: 130px;
+            width: 64px;
+            padding-right: 90px;
+          }
+
+          &::after {
+            content: "+";
+            font-size: 40px;
+            width: 35px;
+            height: 35px;
+          }
+        }
+
+        .active::after {
+          content: "-";
+          font-size: 40px;
+        }
+      }
+    }
+
+    .skout-glossary {
+      .skout-glossary-img-wrapper {
+        padding: 0 135px;
+        .skout-glossary-img {
+          height: 41px;
+          margin: 19.5px 0 50px 0;
+        }
+      }
+
+      .skout-glossary-slider {
+        padding: 60px 0;
+        display: flex;
+
+        .skout-glossary-slider-content {
+          .skout-glossary-slider-item {
+            margin-left: 130px;
+
+            .skout-glossary-slider-item-title {
+              margin: 0 0 73px 0;
+              font-size: 90px;
+              line-height: 120px;
+            }
+
+            .skout-glossary-slider-item-text {
+              margin-left: 70px;
+              padding-left: 36px;
+              max-width: 462.5px;
+              font-size: 24px;
+              line-height: 36px;
+            }
+          }
+        }
+
+        .skout-glossary-slider-menu {
+          width: 420px;
+          margin: 0;
+          height: inherit;
+
+          .skout-glossary-slider-menu-selectors {
+            width: 331px;
+            .skout-glossary-slider-menu-selector {
+              .skout-glossary-slider-menu-line {
+                width: 105px;
+              }
+            }
+          }
+        }
+      }
+    }
+
+    .skout-faqs {
+      padding: 0 135px;
+
+      .skout-faqs-more-questions-btn {
+        margin: 50px auto 102px auto;
+      }
+
+      .skout-faqs-img {
+        height: 32px;
+        margin: 19.5px 0 100px 0;
+      }
+
+      .skout-faqs-accordion {
+        .skout-faqs-accordion-btn {
+          height: 196px;
+
+          .skout-faqs-q-letter {
+            font-size: 110px;
+            line-height: 130px;
+          }
+
+          .skout-faqs-question {
+            font-size: 24px;
+            line-height: 36px;
+            margin-left: 100px;
+          }
+        }
+
+        .skout-faqs-answer {
+          .skout-faqs-answer-wrapper {
+            .skout-faqs-a-letter {
+              font-size: 110px;
+              line-height: 130px;
+            }
```

Comment:
<!--StartFragment--> 
font-size: 110px;    
line-height: 130px;
<!--EndFragment-->
code duplicate 
GLOBAL: keep such values in classes or mixins